scaffolds/java/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use this image as a base
+FROM niaquinto/gradle
+MAINTAINER @toch
+
+# In case someone loses the Dockerfile
+RUN rm -rf /etc/Dockerfile
+ADD Dockerfile /etc/Dockerfile
```

Comment:
I ask because other than that, it seems to me a Dockerfile is not necessary actually (using the base image should work)